site/docs/0.4.0-incubating/rest-api.html

```diff
@@ -551,12 +551,12 @@ <h4 id="request-parameters-3">Request Parameters</h4>
   <tr>
     <td>from</td>
     <td>Offset</td>
-    <td>Offset</td>
+    <td>int</td>
```

Comment:
We fixed these in the Livy Docs (what this is built from), but it won't propagate until 0.4.1 so I'm ok updating this manually here for now, @jerryshao opinion?

Reply:
LGTM.